src/main/build/debug.c

```diff
@@ -63,7 +63,6 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "RANGEFINDER",
     "RANGEFINDER_QUALITY",
     "LIDAR_TF",
-    "CORE_TEMP",
```

Comment:
Removing this will break blackbox logging - maybe it's easiest in this case to rename it and reorder the values so that `coreTemperature` stays at 0.

Reply:
Good find !!! I was completely unaware of that.